mandatory/4-tax.js

```diff
@@ -5,7 +5,9 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+let productTax = (productPrice*20) / 100 + productPrice;
```

Comment:
I'm not sure if you were introduced to `let` vs `const` yet. This is a variable that doesn't need to change, so you'd be better off using `const`. See here: https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.